static_frame/core/index_hierarchy.py

```diff
@@ -808,53 +1041,44 @@ def ndim(self) -> int:
         return self._NDIM
 
     @property
-    def size(self) -> int:
+    def size(self: IH) -> int:
         '''
         Return the size of the underlying NumPy array.
 
         Returns:
             :obj:`int`
         '''
-        if self._recache:
-            return self._levels.__len__() * self._levels.depth
         return self._blocks.size
 
     @property
-    def nbytes(self) -> int:
+    def nbytes(self: IH) -> int:
         '''
         Return the total bytes of the underlying NumPy array.
 
         Returns:
             :obj:`int`
         '''
-        if self._recache:
-            self._update_array_cache()
         return self._blocks.nbytes
```

Comment:
Should we include `nbytes` from other arrays managed in the new `IndexHierarchy`?

Reply:
That seems reasonable. I'll add it!